modules/tinymce/src/plugins/table/main/ts/selection/SelectionTargets.ts

```diff
@@ -122,6 +123,26 @@ export const getSelectionTargets = (editor: Editor, selections: Selections): Sel
     };
   };
 
+  const onSetupWithToggle = (api: Toolbar.ToolbarToggleButtonInstanceApi, isDisabled: (targets: RunOperation.CombinedTargets) => boolean, isActive: (targets: RunOperation.CombinedTargets) => boolean) => {
```

Comment:
Forgot to address this sorry... this duplicates ~50% of the `onSetup` function so please restructure this to remove the duplication.

- [ ] Fix duplication